### PR TITLE
Feature news style & scaffold

### PR DESCRIPTION
**Pull request by varffs**
- adds shuffle item style
- adds shuffle js
- adds style and markup for dotted dividers
- doesnt have drawer and overlay news functionality

### CODE REVIEW

**Comment by mikeyrayvon** [comment]
mmm im a lil stuck on this.  do i have to make a page and set as Posts page?

**Comment by varffs** [comment]
No it is just the index page. Just add some posts

File: index.php [hunk 1]
```diff
@@ -4,22 +4,23 @@
...
+    <div id="shuffle-preloader"></div>
+    <div id="shuffle-container" class="grid-row">
 
 <?php
 if( have_posts() ) {
   while( have_posts() ) {
     the_post();
 ?>
-        <article <?php post_class('grid-item item-s-12 item-m-4'); ?> id="post-<?php the_ID(); ?>">
+        <article <?php post_class('shuffle-item item-s-12 item-m-4'); ?> id="post-<?php the_ID(); ?>">
```
Comment (mikeyrayvon) [inline index.php]: line length is a bit weird at item-m-4 tablet container. maybe`item-m-6 item-l-4`  ?

File: js/main.js [hunk 2]
```diff
@@ -31,4 +33,41 @@ Site = {
...
+    imagesLoaded(_this.shuffleContainer, function() {
+      _this.initShuffle();
+    });
+  },
+
+  initShuffle: function() {
+    var _this = this;
+
+    $('#shuffle-preloader').remove();
+    $(_this.shuffleContainer).css('opacity', 1)
```
Comment (mikeyrayvon) [inline js/main.js]: I wonder if this also works for the filter and ms props that autoprefixer generates for `opacity` on line `755` and `756` of `site.css`. perhaps its better to add remove a `hide` class than to change css props with js

File: index.php (hunk 1 above)
Comment (varffs) [inline index.php]: thats fine by me. I need to clarify how exactly the mobile adaption is supposed to happen as I only remember a kinda throwaway line about developer does mobile or responsive or something like that

File: js/main.js (hunk 2 above)
Comment (varffs) [inline js/main.js]: http://caniuse.com/#search=opacity coverage for everything other than IE8 so thats fine. imo a problem with the 5 versions autoprefixer, we should probably try to refine that so we can catch those safaris but ignore old IE

Comment (mikeyrayvon) [inline js/main.js]: ok. but isnt it better practice to add/remove classes than change fixed props with jquery?

File: index.php (hunk 1 above)
Comment (varffs) [inline index.php]: fine by me as in fine to change the medium breakpoint to 6 col

**Pull request merged**